ui/apps/platform/src/Containers/Clusters/ClustersSidePanel.tsx

```diff
@@ -256,6 +257,7 @@ function ClustersSidePanel({ selectedClusterId, setSelectedClusterId }) {
                     */
                     // TODO After saveCluster returns response without normalize,
                     // something like the preceding commented lines should replace the following:
+                    analyticsTrack('New Cluster');
```

Comment:
This will need to be adapted once this PR is merged.
https://github.com/stackrox/stackrox/pull/4544